src/stores/inbox.js

```diff
@@ -186,6 +186,7 @@ let appState = {
     let body = {
       message_text: message,
       media_url: imageUrl,
+      force: true
```

Comment:
Should this be an attribute we pass into the component instead, so its not always forced?